process/process.go

```diff
@@ -33,15 +34,27 @@ func (w *Process) Validate() error {
 		return err
 	}
 	for _, node := range w.Nodes {
-		mapNode := node.GetMap()
-		if mapNode != nil {
-			for _, output := range mapNode.Outputs {
+		switch n := node.GetType().(type) {
+		case *Process_Node_Map_:
+			for _, output := range n.Map.Outputs {
 				if ref := output.GetRef(); ref != nil {
 					if _, err := w.FindNode(ref.NodeKey); err != nil {
 						return err
 					}
 				}
 			}
+		case *Process_Node_Filter_:
+			for _, condition := range n.Filter.Conditions {
+				switch condition.Predicate {
+				case Process_Node_Filter_Condition_GT,
```

Comment:
[not blocking] would be nice to also add the check for the contain

Reply:
I cannot add a check for contains as I wrote in the PR's description:
> Cannot add process validation because the task's output or event's data should be of type List, not the value in the filter

Here, I don't have access to the data.

Reply:
done